community.dbuild

```diff
@@ -984,4 +989,60 @@ build += {
     extra.exclude: ["plugin"]  // we never build sbt plugins
   }
 
+  // forked to remove the usual bintray gunk
+  ${vars.base} {
+    name: "scalameta"
+    uri:  ${vars.uris.scalameta-uri}
+    // else, bintray stuff goes boom
+    extra.options: ["-Dsbt.prohibit.publish=true"]
+    extra.exclude: ["readme"]  // requires scalatex-site which requires Scala.js
+  }
+
+  // forked to remove wartremover and coursier
+  ${vars.base} {
+    name: "scalafix"
+    uri:  ${vars.uris.scalafix-uri}
+    extra.exclude: [
+      "scalafix-sbt"  // we never build sbt plugins
+      "scalafix-nsc"  // depends on circe which we don't have yet
+      // this didn't work; I decided to postpone any investigation. dbuild output an
+      // immensely long error (tens of megabytes), enough to choke Jenkins
+      "scalafix-tests"
```

Comment:
Yes, I would exclude this directory. Here's where I run rewrites against a mini scalafix community-build. This involves cloning other projects and compiling them.